backend/api/temp/quiz_bash.json

```diff
@@ -0,0 +1,136 @@
+[
+    {
+        "id": 644,
+        "question": "How can you set the default rwx permission to all users on every file which is created in the current shell?",
+        "description": null,
+        "answers": {
+            "answer_a": "umask  0777",
+            "answer_b": "chmod  777",
+            "answer_c": "chown  777",
+            "answer_d": "umask  0666",
+            "answer_e": null,
+            "answer_f": null
+        },
+        "multiple_correct_answers": "false",
+        "correct_answers": {
+            "answer_a_correct": "true",
+            "answer_b_correct": "false",
+            "answer_c_correct": "false",
+            "answer_d_correct": "false",
+            "answer_e_correct": "false",
+            "answer_f_correct": "false"
+        },
+        "correct_answer": "answer_a",
+        "explanation": null,
+        "tip": null,
+        "tags": [
+            {
+                "name": "BASH"
+            }
+        ],
+        "category": "bash",
+        "difficulty": "Easy"
+    },
+    {
+        "id": 9,
+        "question": "#!\/bin\/bash is commonly called as",
+        "description": null,
+        "answers": {
+            "answer_a": "shebang",
+            "answer_b": "hashbang",
+            "answer_c": "Script Initialiser",
+            "answer_d": "None of the Above",
+            "answer_e": null,
```

Comment:
Do we really need to make them `null`? Let's just remove those fields. inexistence should imply `null`

Reply:
You're right.
I haven't made any changes to the format or values yet, they are still in the state I took fetched them.